include/seqan3/alphabet/quality/concept.hpp

```diff
@@ -109,8 +109,8 @@ concept bool quality_concept = requires(q quality)
     //! requires fulfillment of alphabet concept
     requires alphabet_concept<q>;
 
-    //! requires additionally public getter and setter for integral phred score representation
-    { from_phred(quality, typename q::integral_type{}) } -> q;
+    //! requires additionally public getter and setter for rank phred score representation
+    { assign_phred(quality, typename q::rank_type{}) } -> q;
```

Comment:
Why the Metafunctions? If `rank_type` is depend on `q`, why not directly use `q.assign_phred()`?